src/lib/video/index.ts

```diff
@@ -39,6 +41,12 @@ export function createVideoBlock(document: Document, video: any, index: any): HT
     const id = 'VIDEO' + index;
     videoContainerDiv.setAttribute('id', id);
     videoContainerDiv.classList.add('video-container');
+
+    let videoUrl = video.onlineUrl;
+    if (type === VideoType.YouTube && !config.mainFeatures['video-youtube-related-same-channel']) {
```

Comment:
you know, I think I would change and incorporate it with the switch that is just to follow.  Make case VideoType.YouTube:
if (!config.mainFeatures['video-youtube-related-same-channel'] {
videoUrl +=  etc
}
videoContainerDiv.classList.add ....
break;
case VideoType.mp4:
etc